docs/mqtt.md

```diff
@@ -594,6 +594,31 @@ to reconnect to switch to a new virtual host.
 is considered more specific than the port-to-vhost mapping with the `mqtt_port_to_vhost_mapping`
 global parameter and so takes precedence over it.
 
+#### Use client_id from client certificate to authenticate 
+
```

Comment:
Please change the title to: Use the client_id from the Client Certificate to Authenticate

I also added capitalization to some of the words in the heading, please check other sub section titles in OSS RabbitMQ topics, if they use capitalisation for 2nd/3rd words in titles then please keep it consistent here, if they don't then you can leave as is, thanks.

Reply:
umm. but the outer title at level 3 does not have that type of capitalization ... `Authentication with TLS/x509 client certificates` ... ??

Reply:
ok if the others don't have it, leave as it but include the additional words I suggested, thanks